lib/rucio/core/transfer.py

```diff
@@ -1073,7 +1073,7 @@ def __init__(self) -> None:
             self.files_done = 0
             self.files_failed = 0
 
-        def incorporate_stat(self, stat: "Mapping[str, int]") -> None:
+        def incorporate_stat(self, stat: dict[str, int]) -> None:
```

Comment:
same as the previous comment - this method can work just fine with a `Mapping` and does not specifically need a `dict`